tox.ini

```diff
@@ -23,7 +23,9 @@ toxworkdir={homedir}/edxapp_toxenv
 # and look in site-packages.
 usedevelop=True
 setenv =
-    PYTHONHASHSEED = 0
+    # tox will install the appropriate Python packages for us
```

Comment:
Nit: Could this comment have a short `instruct paver not to install the prereqs` or the like?

Reply:
Will reword